### PR TITLE
Refs CORE-5412

### PR DESCRIPTION
**Pull request by rozap**
* added new types to SoqlRep and RowLogCodecs
* change migrate loglevel to supress some spam
* todo: update dependencies file and remove snapshot ref

### CODE REVIEW

File: coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/migration/Migration.scala [hunk 1]
```diff
@@ -26,7 +27,7 @@ object Migration {
                 operation: MigrationOperation = MigrationOperation.Migrate,
                 numChanges: Int = 1,
                 changeLogPath: String = MigrationScriptPath) {
-
+    LogFactory.getLogger().setLogLevel("warning")
```
Comment (urmilan) [inline coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/migration/Migration.scala]: What's this for?

Comment (urmilan) [inline coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/migration/Migration.scala]: Nvm, just saw the explanation in the PR comment.

Comment (chitang) [inline coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/migration/Migration.scala]: I like to cut down logging.  But I find it helpful to have INFO in migration.  Feel better w/o going to check the database if and which migration is done.

Comment (rozap) [inline coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/migration/Migration.scala]: sounds good, i'll remove those

**Comment by rozap** [comment]
google isn't helping much. is there a way to specify the loglevel based on environment? for example, running `sbt test` is a test environment, so it would be helpful to bump up the log level so test output is shown, whereas in a regular dev environment the loglevel would be lower.

**Comment by velvia** [comment]
Specifying the log level based on env is typically done by changing the log
configuration.  For a lot of our services the log config comes from the
Typesafe config file.  That way, you can use deploy tools to change what
logging config is used.

For other services that rely on log4j, you'd just modify or deploy a
different log4j.properties.

On Fri, May 15, 2015 at 3:48 PM, Chris Duranti <notifications@github.com>
wrote:

> google isn't helping much. is there a way to specify the loglevel based on
> environment? for example, running sbt test is a test environment, so it
> would be helpful to bump up the log level so test output is shown, whereas
> in a regular dev environment the loglevel would be lower.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/socrata-platform/data-coordinator/pull/50#issuecomment-102542356>
> .
>



-- 
Evan Chan   |   Principal Engineer, Socrata     |    @evanfchan
*Socrata <http://www.socrata.com/> *| The Data Experience Company™
*Socrata isn't about worldly success, but about changing the world
successfully. This is why I'm excited to be here.*
*What does Socrata do? <https://www.youtube.com/watch?v=yyz86MlkriQ> *(watch
our 2 min video)

**Comment by urmilan** [comment]
If you've resolved @velvia and @chitang's concerns, LGTM. Remember to replace snapshot dependencies before merging

**Pull request merged**